src/main/java/de/chojo/repbot/data/updater/SqlUpdater.java

```diff
@@ -14,31 +15,39 @@
 import static org.slf4j.LoggerFactory.getLogger;
 
 public class SqlUpdater {
-    public static final int MAJOR = 1;
-    public static final int PATCH = 9;
+    private final int major;
+    private final int patch;
     private static final Logger log = getLogger(SqlUpdater.class);
     private final DataSource source;
     private final String versionTable;
     private final String[] schemas;
     private final QueryReplacement[] replacements;
 
-    private SqlUpdater(DataSource source, String versionTable, QueryReplacement[] replacements, String[] schemas) {
+    private SqlUpdater(DataSource source, String versionTable, QueryReplacement[] replacements, String[] schemas, Pair<Integer, Integer> version) {
         this.source = source;
         this.versionTable = versionTable;
         this.replacements = replacements;
         this.schemas = schemas;
+        major = version.first;
+        patch = version.second;
     }
 
-    public static SqlUpdaterBuilder builder(DataSource dataSource) {
-        return new SqlUpdaterBuilder(dataSource);
+    public static SqlUpdaterBuilder builder(DataSource dataSource) throws IOException {
+        var version = "";
+        try (var in = Info.class.getClassLoader().getResourceAsStream("database/version")) {
+            version = new String(in.readAllBytes()).trim();
+        }
+
+        String[] ver = version.split("\\.");
+        return new SqlUpdaterBuilder(dataSource, Pair.of(Integer.valueOf(ver[0]), Integer.valueOf(ver[1])));
```

Comment:
```suggestion
        return new SqlUpdaterBuilder(dataSource, Pair.of(Integer.parseInt(ver[0]), Integer.parseInt(ver[1])));
```

Reply:
I dont see an advance here :thinking:
Both method behave exactly the same. parseInt is just when you want to add an radix or if you want to allow stuff like - or +, which I dont want as well.

Reply:
The method just does unnecessary boxing.
![image](https://user-images.githubusercontent.com/39954661/124600554-21808280-de67-11eb-9115-d3bae7c68555.png)

According to the docs, this would be the same as `new Integer(Integer.parseInt(someString))`

Reply:
Well alright. Actually just read the documentation and didnt looked into the code.